WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -161,6 +175,27 @@ private extension StoreOnboardingViewModel {
         defaults[.completedAllStoreOnboardingTasks] = true
     }
 
+    @MainActor
+    func isFreeTrialPlan() async -> Bool {
```

Comment:
❓ I was wondering if it makes sense to enable the check behind a feature flag in case we want to coordinate the free trial launch with Eagle. In `trunk`, the free trial feature flag is still disabled in production.

Reply:
Good point.

The Free trial feature has been released in [this PR](https://github.com/woocommerce/woocommerce-ios/pull/9265). I think the feature flag is not removed to use for future iterations. 

So, we don't have to keep this change behind a feature flag.